packages/angular/build/src/tools/esbuild/javascript-transformer-worker.ts

```diff
@@ -27,6 +27,25 @@ interface JavaScriptTransformRequest {
 const textDecoder = new TextDecoder();
 const textEncoder = new TextEncoder();
 
+/**
+ * The current list of Angular partial compilation declaration function names.
+ * Used to determine if linking of a JavaScript file is required.
+ * If any additional declarations are added or otherwise changed, this list
+ * MUST be updated.
+ * The component declaration is first as it is the most likely to be present.
+ */
+const LINKER_DECLARATION_NAMES = [
```

Comment:
WDYT of reducing this to a single search for just `ɵɵngDeclare`, which I expect is sufficiently unique to avoid false positives and reduces the maintenance/syncing cost of having the check in the CLI.